blog/_posts/2024-11-15-dimensionality-reduction-for-flu-and-sars-cov-2.md

```diff
@@ -0,0 +1,171 @@
+  - make two-dimensional maps from mutation data using methods with exciting names like principal components analysis (PCA), multidimensional scaling (MDS), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP)
+  - plot trees and maps in a single interactive figure that allowed us to highlight bits of the tree or a map and see the same viruses in the other parts of the figure
+
+By August 2019, Sravani had made the prototype below (Figure 4) from mutations in a type of seasonal flu called “H3N2” which causes the most hospitalizations and deaths each year.
+
+<figure>
+    <img src="/images/blog/cartography-final-internship-prototype.png" alt="Sravani's final internship prototype showing maps based on flu mutations" />
+    <figcaption><strong>Figure 4. Static view of Sravani's final internship prototype showing individual viruses in a family tree (top left) and corresponding positions of the same viruses in maps based on flu mutations including PCA (top middle), MDS (top right), t-SNE (bottom left), and UMAP (bottom right).</strong> Viruses in two specific groups from the tree (blue and orange) have been selected to show how their placement in the tree compares to their placement in the maps.</figcaption>
+</figure>
+
+The prototype revealed some interesting patterns:
+
+  1. Most of the maps placed pairs of viruses with the same mutations closer together than pairs with different mutations.
+  2. Some of the maps (like MDS's) actually acted like a real map with the distance between viruses on the map matching exactly the number of mutations that differed between those viruses.
+  3. Other maps (like t-SNE's) didn't act like real maps, but they tightly clustered similar viruses into groups in the same space where we could easily find those groups by eye.
+  4. The groups of viruses in these maps often matched the groups we had already defined in the tree.
+
+Sravani and I were excited enough about these results to agree that we should keep this project going a little longer.
+In October 2019, we decided to meet once a month while Sravani refined the prototype above and drafted a short summary of the results in the form of a scientific paper that we could post online somewhere.
+
+### Can we find groups of related flu viruses when we _can't_ build a family tree?
+
+Sravani and I continued to meet monthly through the beginning of the SARS-CoV-2 pandemic, she learned how to write a scientific paper, wrote the first full draft of a paper, and referenced this work in her college applications.
+By June 2023, we'd both been busy with other projects.
+Sravani had been focused on class work as an undergraduate in the University of Washington's Computer Science program.
+I had been working with the Nextstrain team on pandemic response efforts.
+By the summer of 2023, Sravani was eager to revise the original paper and publish it in a scientific journal.
```

Comment:
I don't have a great suggestion for replacement, but the repeated "By {time period}," with just two intervening sentences, is clunky.

Reply:
I felt this, too, as I wrote it. I just replaced that "By the summer of 2023," to make sentence read:

> Despite our other commitments, Sravani was eager to revise the original paper and publish it in a scientific journal.